lib/ruby_ui/switch/switch.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module RubyUI
+  class Switch < Base
+    def view_template
+      attrs => { include_hidden:, **input_attrs }
```

Comment:
I think we can move this to initializer and allow to pass the checked and unchecked values, what do you think?
```ruby
def initialize(include_hidden: true, checked_value: "1", unchecked_value: "0", **attrs)
  @include_hidden = include_hidden
  @checked_value = checked_value
  @unchecked_value = unchecked_value
  super(**attrs)
end
```